HerPublicWebsite.BusinessLogic/ExternalServices/EmailSending/GovUkNotifyApi.cs

```diff
@@ -48,14 +48,14 @@ private void SendEmail(GovUkNotifyEmailModel emailModel)
             }
         }
 
-        public void SendReferenceCodeEmail
+        private void SendReferenceCodeEmail
```

Comment:
Private methods should be moved to the bottom of the file, under all the public ones.